src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Media/Given_ImageBrushWriteableBitmap.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using NUnit.Framework;
+using SamplesApp.UITests.TestFramework;
+using Uno.UI.RuntimeTests.Extensions;
+using Uno.UI.RuntimeTests.Helpers;
+using Windows.Foundation.Metadata;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Media.Imaging;
+using static Private.Infrastructure.TestServices;
+
+namespace Uno.UI.RuntimeTests.Tests.Windows_UI_Xaml_Media
+{
+	[TestClass]
+	[RunsOnUIThread]
+	public partial class ImageBrush_WriteableBitmap
+	{
+		[TestMethod]
+		public async Task When_UseWriteableBitmapAsImageBrushSource()
+		{
+#if __MACOS__  
+			[Ignore("That test cause a Runtime error with MACOS")]
+#endif
```

Comment:
This should be above test method declaration (ie, above the line `public async Task When_UseWriteableBitmapAsImageBrushSource`)